package.json

```diff
@@ -106,7 +106,7 @@
     "eslint-config-airbnb-typescript": "^14.0.1",
     "eslint-plugin-eslint-comments": "^3.0.1",
     "eslint-plugin-import": "2.22.1",
-    "eslint-plugin-jsdoc": "^22.1.0",
+    "eslint-plugin-jsdoc": "^38.1.6",
```

Comment:
Updated to be compatible with npm v8.6

Reply:
I think this is fine. If it was a production package that might be different but it seems OK.